src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+// eslint-disable-next-line max-len, prettier/prettier
+import { TodoCompletedCategory as CompletedCategory } from '../../types/TodoCompletedCategory';
+
+type Props = {
+  countOfNotCompletedTodos: number;
+  completedCategory: CompletedCategory;
+  onCompletedCategory: React.Dispatch<React.SetStateAction<CompletedCategory>>;
+};
+
+const categoryForNav = {
```

Comment:
activityFilters?cuz I don't understand exactly which categories are needed for navigation
```suggestion
const activityFilters = {
```